packages/commons/src/nfc-reader.class.ts

```diff
@@ -0,0 +1,19 @@
+import NFCTag from './nfc-tag.class';
+
+export default class NFCReader {
+  private static instance: NFCReader;
+
+  currentTag: NFCTag | undefined;
+
+  private constructor(_currentTag?: NFCTag) {
+    this.currentTag = _currentTag;
+  }
+
+  public static getInstance(): NFCReader {
+    if (!NFCReader.instance) {
+      NFCReader.instance = new NFCReader();
+    }
+
+    return NFCReader.instance;
+  }
```

Comment:
I don't think that we will need / use a singleton here.

```suggestion
```

Reply:
True

Reply:
resolved by 7f6fca830e15635acf95b520a301f99af5847662